CPT331.Core/Extensions/StringExtensions.cs

```diff
@@ -8,24 +8,39 @@
 
 namespace CPT331.Core.Extensions
 {
+	/// <summary>
+	/// Provides System.String type extensions.
+	/// </summary>
 	public static class StringExtensions
 	{
-		private static SHA1CryptoServiceProvider _sha1CryptoServiceProvider;
```

Comment:
Decided that this didn't need to live the potentially the life of the application.